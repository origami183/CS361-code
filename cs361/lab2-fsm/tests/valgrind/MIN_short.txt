==647209== Memcheck, a memory error detector
==647209== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==647209== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==647209== Command: ../fsm 0 1 5
==647209== 
NEW.ADMIT -> RDY
RDY.DISPATCH -> RUN
RUN.EXIT -> TRM
==647209== 
==647209== HEAP SUMMARY:
==647209==     in use at exit: 0 bytes in 0 blocks
==647209==   total heap usage: 2 allocs, 2 frees, 8,224 bytes allocated
==647209== 
==647209== All heap blocks were freed -- no leaks are possible
==647209== 
==647209== For lists of detected and suppressed errors, rerun with: -s
==647209== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
