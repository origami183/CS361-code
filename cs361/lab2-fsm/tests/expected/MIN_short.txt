NEW.ADMIT -> RDY
  [Process is live]
RDY.DISPATCH -> RUN
RUN.EXIT -> TRM

  Process stats:
    runtime: 1
    normal exit: true
